Description: Disble_global_leaks_check
 Disable test to check for global leaks
--- a/test/common/index.js
+++ b/test/common/index.js
@@ -491,7 +491,7 @@
 exports.leakedGlobals = leakedGlobals;
 
 // Turn this off if the test should not check for global leaks.
-exports.globalCheck = true;
+// exports.globalCheck = true;
 
 process.on('exit', function () {
   if (!exports.globalCheck) return;
@@ -984,4 +984,4 @@
     e.syscall = syscall;
     return e;
   };
-}
\ No newline at end of file
+}
